doc/source/whatsnew/v1.6.0.rst

```diff
@@ -229,6 +229,7 @@ MultiIndex
 I/O
 ^^^
 - Bug in :func:`read_sas` caused fragmentation of :class:`DataFrame` and raised :class:`.errors.PerformanceWarning` (:issue:`48595`)
+- Regression in :class:`StataReader` caused all files to needlessly be buffered in memory (:issue:`48922`)
```

Comment:
Could you move this in the `Performance improvement` section? Like `Memory improvement in ...`